src/views/artefact-editor/artefact-editor.vue

```diff
@@ -509,25 +508,32 @@ export default class ArtefactEditor
             parseInt(this.$route.params.editionId),
             parseInt(this.$route.params.artefactId)
         );
-        const imagedObject = this.$state.imagedObjects.find(
-            this.artefact.imagedObjectId
-        );
-        if (!imagedObject) {
-            throw new Error(
-                `Can't find imaged object ${this.artefact.imagedObjectId} belonging to artefact ${this.artefact.id}`
-            );
-        }
-        this.imageStack =
-            this.artefact.side === 'recto'
-                ? imagedObject.recto
-                : imagedObject.verso;
-        if (!this.imageStack) {
-            throw new Error(
-                `ImagedObject ${this.artefact.imagedObjectId} doesn't contain the ` +
-                    `${this.artefact.side} side even though artefact ${this.artefact.id} references it`
+
+        if (!this.artefact.isVirtual) {
+            const imagedObject = this.$state.imagedObjects.find(
+                this.artefact.imagedObjectId
             );
+            if (!imagedObject) {
+                throw new Error(
```

Comment:
See above.

Reply:
We have the `prepare` methods of in our internal state, that should make sure we have this already. If we don't, something is really messed up.